<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Teacher&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="teacher">
         <attribute name="title">
          <string>Teacher</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableWidget" name="teacherTableWidget">
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>100</number>
              </attribute>
              <column>
               <property name="text">
                <string>TeacherID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="teacherTimeTableWidget">
              <property name="maximumSize">
               <size>
                <width>401</width>
                <height>16777215</height>
               </size>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <row>
               <property name="text">
                <string>Mon</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Tun</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Wed</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Thu</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Fri</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Sat</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>3</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>4</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>5</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>6</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>7</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>8</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Class">
         <attribute name="title">
          <string>Class</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="1">
           <widget class="QTableWidget" name="gradeTableWidget">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>ClassID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Grade</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
          <item row="0" column="2">
           <widget class="QTableWidget" name="gradeDetailsTableWidget">
            <property name="columnCount">
             <number>3</number>
            </property>
            <column>
             <property name="text">
              <string>Subject</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Teacher</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Hours</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="1" column="0" alignment="Qt::AlignTop">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Schedule">
         <attribute name="title">
          <string>Schedule</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTabWidget" name="scheduleTabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="mon">
               <attribute name="title">
                <string>Mon</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="monSchedTableWidget">
                  <column>
                   <property name="text">
                    <string>Lesson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tue">
               <attribute name="title">
                <string>Tue</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="tueSchedTableWidget">
                  <property name="columnCount">
                   <number>3</number>
                  </property>
                  <column>
                   <property name="text">
                    <string>Leson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="wed">
               <attribute name="title">
                <string>Wed</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="wedSchedTableWidget">
                  <column>
                   <property name="text">
                    <string>Leson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="thu">
               <attribute name="title">
                <string>Thu</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="thuSchedTableWidget">
                  <column>
                   <property name="text">
                    <string>Leson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="fri">
               <attribute name="title">
                <string>Fri</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="friSchedTableWidget">
                  <column>
                   <property name="text">
                    <string>Leson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="sat">
               <attribute name="title">
                <string>Sat</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QTableWidget" name="satSchedTableWidget">
                  <column>
                   <property name="text">
                    <string>Leson</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Subject</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Teacher</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QTableWidget" name="scheduleTableWidget">
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>70</number>
            </attribute>
            <column>
             <property name="text">
              <string>Grade</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="3">
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QPushButton" name="generateButton">
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuSchedule"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
